.github/workflows/ci.yaml

```
@@ -69,14 +72,36 @@ jobs:
         with:
           name: coverage-xml
       - name: Fix coverage file for sonarcloud
-        run: sed -i "s/home\/runner\/work\/starlite-jwt\/starlite_jwt/github\/workspace/g" coverage.xml
+        run: sed -i "s/home\/runner\/work\/starlite\/starlite/github\/workspace/g" coverage.xml
```

this should be reverted:

```suggestion
        run: sed -i "s/home\/runner\/work\/starlite-jwt\/starlite_jwt/github\/workspace/g" coverage.xml
```

---

Done